SmartDeviceLink/SDLScreenManager.h

```diff
@@ -147,13 +209,61 @@ typedef void(^SDLScreenManagerUpdateCompletionHandler)(NSError *__nullable error
  */
 - (void)endUpdatesWithCompletionHandler:(nullable SDLScreenManagerUpdateCompletionHandler)handler;
 
+#pragma mark Soft Button
+
+- (nullable SDLSoftButtonObject *)softButtonObjectNamed:(NSString *)name;
+
+#pragma mark Choice Sets
+
 /**
- Find a current soft button object with a specific name
+ Preload cells to the head unit. This will *greatly* reduce the time taken to present a choice set. Any already matching a choice already on the head unit will be ignored. You *do not* need to wait until the completion handler is called to present a choice set containing choices being loaded. The choice set will wait until the preload completes and then immediately present.
 
- @param name The name of the soft button object to find
- @return The soft button object or nil if there isn't one with that name
+ @param choices The choices to be preloaded.
+ @param handler The handler to be called when complete.
  */
-- (nullable SDLSoftButtonObject *)softButtonObjectNamed:(NSString *)name;
+- (void)preloadChoices:(NSArray<SDLChoiceCell *> *)choices withCompletionHandler:(nullable SDLPreloadChoiceCompletionHandler)handler;
+
+/**
+ Delete loaded cells from the head unit. If the cells don't exist on the head unit they will be ignored.
+
+ @param choices The choices to be deleted. These will be matched via a hash of the text, image name(s), and VR commands.
+ */
+- (void)deleteChoices:(NSArray<SDLChoiceCell *> *)choices;
```

Comment:
When I go to delete choices every single choice is given an `interactionChoiceSetID` of 65535, even if they have different text, images, etc..

Reply:
Should be fixed